benchbuild/source/base.py

```diff
@@ -0,0 +1,125 @@
+"""
+Provide a base interface for downloadable sources.
+"""
+import abc
+import itertools
+from typing import Iterable, List, Mapping, Union
+
+import attr
+
+from benchbuild import variants
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'variants' in module 'benchbuild'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)